Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangularFEMForceField.inl

```diff
@@ -359,6 +359,93 @@ SReal TriangularFEMForceField<DataTypes>::getPotentialEnergy(const core::Mechani
     return 0;
 }
 
+template <class DataTypes>
+void TriangularFEMForceField<DataTypes>::computeElementStiffnessMatrix(type::Mat<9, 9, typename TriangularFEMForceField<DataTypes>::Real>& S, type::Mat<9, 9, typename TriangularFEMForceField<DataTypes>::Real>& SR, const MaterialStiffness& K, const StrainDisplacement& J, const Transformation& Rot)
+{
+    type::MatNoInit<3, 6, Real> Jt;
+    Jt.transpose(J);
+
+    type::MatNoInit<6, 6, Real> JKJt;
+    JKJt = J * K * Jt;  // in-plane stiffness matrix, 6x6
+
+    // stiffness JKJt expanded to 3 dimensions
+    type::Mat<9, 9, Real> Ke; // initialized to 0
+    // for each 2x2 block i,j
+    for (unsigned i = 0; i < 3; i++)
+    {
+        for (unsigned j = 0; j < 3; j++)
+        {
+            // copy the block in the expanded matrix
+            for (unsigned k = 0; k < 2; k++)
+                for (unsigned l = 0; l < 2; l++)
+                    Ke[3 * i + k][3 * j + l] = JKJt[2 * i + k][2 * j + l];
+        }
+    }
+
+    // rotation matrices. TODO: use block-diagonal matrices, more efficient.
+    type::Mat<9, 9, Real> RR, RRt; // initialized to 0
+    for (int i = 0; i < 3; ++i)
+        for (int j = 0; j < 3; ++j)
+        {
+            RR[i][j] = RR[i + 3][j + 3] = RR[i + 6][j + 6] = Rot[i][j];
+            RRt[i][j] = RRt[i + 3][j + 3] = RRt[i + 6][j + 6] = Rot[j][i];
+        }
+
+    S = RR * Ke;
+    SR = S * RRt;
+}
+
+template <class DataTypes>
+void TriangularFEMForceField<DataTypes>::addKToMatrix(sofa::linearalgebra::BaseMatrix* mat, SReal k, unsigned& offset)
```

Comment:
```suggestion
void TriangularFEMForceField<DataTypes>::addKToMatrix(sofa::linearalgebra::BaseMatrix* mat, SReal k, unsigned int& offset)
```

Just a very small change, otherwise VS studio can't find/switch between the implementation/declaration